tokenizer/src/main/scala/anorm/TokenizedStatement.scala

```diff
@@ -77,7 +80,8 @@ private[anorm] object TokenizedStatement {
       }
     } else parts.headOption match {
       case Some(part) =>
-        val it = new StringTokenizer(part, "%", true).asScala.map(_.toString)
+        //val it = new StringTokenizer(part, "%", true).asScala.map(_.toString)
```

Comment:
Perhaps delete this comment as well. The code is still in Git if you need it again in the future.